modyn/supervisor/internal/utils/evaluation_status_tracker.py

```diff
@@ -0,0 +1,39 @@
+from enlighten import Counter, Manager
+
+
```

Comment:
I haven't checked in detail yet - Do you think there is potential to make some inheritance with the status bar classes by Francesco or is there no iheritance possible?

Reply:
(supervisor_counter)

Reply:
I also thought about this (also to only have one class). But IMHO there is not much potential as the evaluation status tracker is pretty simple and created on a per-evaluation-dataset basis. The `supervisor_counter` on the other hand changes states (btw. training and downsampling) during training, has several epochs and is only created once during training-invocation.

Reply:
Ok, makes sense. Then let's keep that seperate.